src/phoneField/index.tsx

```diff
@@ -0,0 +1,179 @@
+import React, { PureComponent } from 'react'
+import cc from 'classcat'
+import prefix from '_utils'
+import isEmpty from 'lodash.isempty'
+import isEqual from 'lodash.isequal'
+import SelectField from 'selectField'
+import TextField from 'textField'
+import countryTelData from 'country-telephone-data'
+import style from 'phoneField/style'
+
+const FIELDS: phoneFields = {
+  PHONENUMBER: 'phoneNumber',
+  PHONEPREFIX: 'phonePrefix',
+}
+
+interface PhoneFieldOnChangeParameters {
+  name: string
+  value: object
```

Comment:
object type is nearly as uninformative as 'any'.
Should we use something more semantic (interface or inline declaration) instead of a catch-all object type?
do we have any guideline about using detailed declarations?

Reply:
I follow the way of the global `OnChangeParameters` interface.
`declare interface OnChangeParameters {
  readonly name: string
  readonly value: string | number | boolean
}`
I think our guidelines is to be more precise as possible.
I don't find a way to be more precise and match with TypeScript validation.

Reply:
Fixed in pair programming